libs/community/langchain_community/agent_toolkits/openapi/planner.py

```diff
@@ -346,6 +385,7 @@ def create_openapi_agent(
     verbose: bool = True,
     agent_executor_kwargs: Optional[Dict[str, Any]] = None,
     allow_dangerous_requests: bool = False,
+    allow_operations: Sequence[Operation] = ('GET', 'POST'),
```

Comment:
```suggestion
    allowed_operations: Sequence[Operation] = ('GET', 'POST'),
```

Reply:
@eyurtsev  Changes have been made to address this issue. Please see [commit](https://github.com/langchain-ai/langchain/pull/22962/commits/762f03d17bb5d55a390b101b4ebda160d8933394).